docs/source/tutorial/first.rst

```diff
@@ -63,33 +78,80 @@ Let us clarify the terminology in Optuna as follows:
 In Optuna, we use the study object to manage optimization. Method :func:`~optuna.study.create_study` returns a study object.
 A study object has useful properties for analyzing the optimization outcome.
 
+To get the best params:
+
 .. code-block:: python
 
-    In [5]: study.best_params
-    Out[5]: {'x': 1.9926578647650126}
+    study.best_params
 
-    In [6]: study.best_value
-    Out[6]: 5.390694980884334e-05
+Out:
 
-    In [7]: study.best_trial
-    Out[7]: FrozenTrial(number=26, state=<TrialState.COMPLETE: 1>, params={'x': 1.9926578647650126}, user_attrs={}, system_attrs={'_number': 26}, value=5.390694980884334e-05, intermediate_values={}, datetime_start=datetime.datetime(2018, 5, 9, 10, 23, 0, 87060), datetime_complete=datetime.datetime(2018, 5, 9, 10, 23, 0, 91010), trial_id=26)
+.. code-block:: console
+
+    {'x': 1.9926578647650126}
+
+To get the best value:
+
+.. code-block:: python
+
+    study.best_value
+
+Out:
+
+.. code-block:: console
+
+    5.390694980884334e-05
+
+To get the best trial:
+
+.. code-block:: python
+
+    study.best_trial
+
+Out:
+
+.. code-block:: console
+
+    FrozenTrial(number=26, state=<TrialState.COMPLETE: 1>, params={'x': 1.9926578647650126}, user_attrs={}, system_attrs={'_number': 26}, value=5.390694980884334e-05, intermediate_values={}, datetime_start=datetime.datetime(2018, 5, 9, 10, 23, 0, 87060), datetime_complete=datetime.datetime(2018, 5, 9, 10, 23, 0, 91010), trial_id=26)
+
+To get all trials:
+
+.. code-block:: python
+
+    study.trials  # all trials
+
+Out:
+
+.. code-block:: console
 
-    In [8]: study.trials  # all trials
-    Out[8]:
     [FrozenTrial(number=0, state=<TrialState.COMPLETE: 1>, params={'x': -4.219801301030433}, user_attrs={}, system_attrs={'_number': 0}, value=38.685928224299865, intermediate_values={}, datetime_start=datetime.datetime(2018, 5, 9, 10, 22, 59, 983824), datetime_complete=datetime.datetime(2018, 5, 9, 10, 22, 59, 984053), trial_id=0),
      ...
      user_attrs={}, system_attrs={'_number': 99}, value=8.2881000286123179, intermediate_values={}, datetime_start=datetime.datetime(2018, 5, 9, 10, 23, 0, 886434), datetime_complete=datetime.datetime(2018, 5, 9, 10, 23, 0, 891347), trial_id=99)]
 
-    In [9]: len(study.trials)
-    Out[9]: 100
+To get the number of trials:
 
+.. code-block:: python
+
+    len(study.trials)
+
+Out:
+
+.. code-block:: console
+
+    100
 
 By executing :func:`~optuna.study.Study.optimize` again, we can continue the optimization.
 
 .. code-block:: python
 
-    In [10]: study.optimize(objective, n_trials=100)
-    ...
+    study.optimize(objective, n_trials=100)
+
+.. code-block:: python
```

Comment:
How about adding some text before this block or merge this block with the previous one? For the former, something along "To get the updated number of trials.".